parts/linux/cloud-init/artifacts/components.json

```diff
@@ -686,16 +703,17 @@
             "versionsV2": [
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.4.54"
+                "latestVersion": "1.4.56",
+                "previousLatestVersion": "1.4.54"
               },
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.5.35",
-                "previousLatestVersion": "1.5.32"
+                "latestVersion": "1.5.38",
+                "previousLatestVersion": "1.5.35"
               },
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.6.10",
+                "latestVersion": "1.6.11",
                 "previousLatestVersion": "1.6.7"
```

Comment:
this should also be `v1.6.10` right?

Reply:
same here. v1.6.10 version was not rolledout to all regions in aks-rp. It was rolled out only to specfic subscriptions in few regions.